doc/bible.md

```diff
@@ -256,7 +256,8 @@ To determine relative participation notary scoring, refer to the following:
 
 - KMD -> LTC Notarisation: 0.0325 
 - Smartchain -> KMD notarisation: 0.8698/(number of main server chains*, not including KMD/LTC)
-- 3rd Party chain -> KMD notarisation: 0.0977/(number of active 3rd party server chains*, not including KMD) 
+- 3rd Party chain -> KMD notarisation: 0.0977/(number of active 3rd party server chains*, not including KMD)
+- MM2 Seed Node -> 0.2 per hour uptime with correct version
```

Comment:
This seems incompatible with any conversations ever had or even the concept of being a notary

Reply:
This is not required for NN operations and optional - i.e. you are free to not run seed node. The dPoW rules remain unchanged. As outlined in the service description it is for further decentralization of DEX network.

Reply:
>This is not required for NN operations and optional 

Tying it to points makes it a requirement.